src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void sayHello() {
         String logo = " ____        _        \n"
```

Comment:
since logo should be a constant, can change the variable name to uppercase and set it as a constant